profiles/coreos/arm64/package.accept_keywords

```diff
@@ -3,6 +3,8 @@
 
 # needed by arm64-native SDK
 app-misc/editor-wrapper *
+=app-emulation/open-vmdk-1.0 *
+=virtual/cdrtools-0 *
```

Comment:
Can you please this line to https://github.com/flatcar-linux/coreos-overlay/blob/47dc477b74443207f10afaa5a56a25dd10efd0e5/profiles/coreos/arm64/package.accept_keywords#L135 , so all entries are sorted?

Reply:
Yes, done